docs/how_to/query.md

```diff
@@ -51,6 +51,38 @@ You will notice that in the setup linked above, M3DB has just one unaggregated n
     all: false
 ```
 
+## ID generation
+
+The default generation scheme for IDs is unfortunately prone to collisions, but is left as is for back-compat reasons. It is suggested to set the ID generation scheme to one of either "quoted" or "prepend_meta". Quoted generation scheme yields the most human-readable IDs, whereas Prepend_Meta is better for more compact IDS, or if tags are expected to contain non-ASCII characters. To set the ID generation scheme, add the following to your coordinator configuration yaml file:
```

Comment:
"but remains the default for backwards compatibility reasons"

also in one case you capitalize prepend_meta and in the other you don't. Use something like `prepend_meta` everywhere and spell it exactly how it should be in the config YAML

Reply:
Good call :+1: